src/auth/jwtclient.ts

```diff
@@ -150,6 +151,29 @@ export class JWT extends OAuth2Client {
     }
   }
 
+  /**
+   * Fetches an ID token.
+   * @param targetAudience the audience for the fetched ID token.
+   */
+  async fetchIdToken(targetAudience: string): Promise<string> {
+    // Create a new gToken for fetching an ID token
+    const gtoken = new GoogleToken({
+      iss: this.email,
+      sub: this.subject,
+      scope: this.scopes,
+      keyFile: this.keyFile,
+      key: this.key,
+      additionalClaims: {target_audience: targetAudience},
+    });
+    const token = await gtoken.getToken({
```

Comment:
What is up with this `const token`? Why aren't you using it afterwards?

Reply:
Removed `const token`!